### ISSUE TITLE
Accessibility - My Incidents Screen - Not Showing My Incidents Information

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/OCIO-DOLSafety-iOS#38]
Since the application is being used by other people when I tapped on the “My Incidents” link/tab, I don’t see any other issues. I would have expected to see some of the other people's information or at least my own testing information and not having a blank main content section of the screen. 

If it is empty, then there should be a message telling the user there is no incident in the system. 

There is a similar issue for the "All Incidents" screen, and the only thing that changed and wasn’t announced to the VoiceOver user is the heading “All Reported Incidents” and the "My Incidents" along with the same for the heading at the top of the screen.

**Comment by aanthonyDOL** [comment]
This issue is a duplicate of another, and has been fixed. The fix was to assign you the right permissions  via Sharepoint.

**Comment by jfc3-dol** [comment]
Is that going to have to be done for all users added to the application in the future, or is this because we are testing the application? Otherwise, that will be a pain for those in charge of the application administration.

Another question would be only certain DOL people able to download the application or can anyone, which they then would need to request access?

**Comment by aanthonyDOL** [comment]
The app will be made available to all DOL Employees with GFE enrolled device. Meaning the device has to be enrolled in Intune. You was given access due to testing, moving forward we will include access to all enrolled devices.

**Comment by jfc3-dol** [comment]
Thanks for the update. Will make sure it works with the next application update.
